src/main/scala/dotvisualizer/dotnodes/ModuleNode.scala

```diff
@@ -12,20 +12,30 @@ import scala.collection.mutable.ArrayBuffer
 case class ModuleNode(
   name: String,
   parentOpt: Option[DotNode],
-  var url_string:Option[String]= Some("TopOfVisualizer.dot.svg"),
+  var url_string:Option[String]= None,
   subModuleDepth: Int = 0
 ) extends DotNode {
 
   val inputs: ArrayBuffer[DotNode] = new ArrayBuffer()
   val outputs: ArrayBuffer[DotNode] = new ArrayBuffer()
   val namedNodes: mutable.HashMap[String, DotNode] = new mutable.HashMap()
   val connections: mutable.HashMap[String, String] = new mutable.HashMap()
+  val biConnections: mutable.HashMap[String, ArrayBuffer[String]] = new mutable.HashMap[String, ArrayBuffer[String]]() {
```

Comment:
Doesn't the term biconnect in Chisel/FIRRTL refer to `<>` which is a different beast?

Reply:
change to Analog connect